apps/chat-ios/README.md

```diff
@@ -0,0 +1,41 @@
+# Amazon Chime SDK iOS Chat Demo
+
+## Overview
+
+By completing the steps below, you will learn how to run the Amazon Chime SDK iOS chat demo applications on your mobile device or on a simulator. This application enables users to send and receive messages/image attachments in chat channels in real time. This application is designed in a way that can be used as a starting point for your own application, or as a quick way to explore the features of the Amazon Chime SDK messaging. Please see the [Build chat applications in iOS and Android with Amazon Chime SDK messaging](https://aws.amazon.com/blogs/business-productivity/build-chat-applications-in-ios-and-android-with-amazon-chime-sdk-messaging) for more information.
```

Comment:
The blog link is broken, could you please update it.

Reply:
The messaging SDK chat demo README calls Amazon Chime SDK for messaging and not Amazon Chime SDK messaging hence suggest to follow the same convention in this README as well as Android chat demo README.

Reply:
> The messaging SDK chat demo README calls Amazon Chime SDK for messaging and not Amazon Chime SDK messaging hence suggest to follow the same convention in this README as well as Android chat demo README.

The official name is Amazon Chime SDK messaging: https://docs.aws.amazon.com/chime/latest/dg/using-the-messaging-sdk.html
This blog post is reviewed by the author of web chat demo blog, Ryan, as well. The README is only a mini version of the blog. So `Amazon Chime SDK messaging` should be fine.

Reply:
> The blog link is broken, could you please update it.

The link will be live once publish, the url has been assigned in WordPress: https://aws.amazon.com/blogs/business-productivity/build-chat-applications-in-ios-and-android-with-amazon-chime-sdk-messaging/